src/components/Image/README.md

```diff
@@ -100,4 +110,8 @@ Supports attributes from [`<img>`](https://developer.mozilla.org/en-US/docs/Web/
 ## Events
 
 Supports events from [`<img>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img).
+
+| Event        | Type | Description                                        |
+| ------------ | ---- | -------------------------------------------------- |
+| image:loaded | -    | Image is loaded and fade in transition is complete |
```

Comment:
maybe it would be more accurate to call it `image:ready` or `image:visible`? But it's not a blocker issue imo